### PR TITLE
Added windows instructions in the docs  quick start

### PR DESCRIPTION
**Pull request by nerdalert**
- Also adds container instructions for network router

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -52,6 +52,25 @@ Query the status of `nexd` and follow the instructions to register your device.
 sudo nexctl nexd status
 ```
 
+### Windows
+
+There are various ways for windows to start processes and services with Powershell and an eventual nexodus agent GUI, but for testing Nexodus on a Windows device the following is a simple method.
```
Comment (russellb) [inline docs/quickstart.md:57]:
```suggestion
There are various ways for Windows to start processes and services with Powershell and an eventual nexodus agent GUI, but for testing Nexodus on a Windows device, the following is a simple method.
```

File: docs/quickstart.md [hunk 2]
```diff
@@ -52,6 +52,25 @@ Query the status of `nexd` and follow the instructions to register your device.
...
+
+- For Windows, you will need to install Wireguard from the official [Windows Wireguard](https://www.wireguard.com/install/) installer.
+- Download the zip file containing the `nexd.exe` and `nexctl.exe` executables. For simplicity, move those two files to `C:\` (or into a directory in your $PATH).
```
Comment (russellb) [inline docs/quickstart.md:60]: Which zip file? Maybe link to it here?

```suggestion
- Download the [zip file](https://nexodus-io.s3.amazonaws.com/qa/nexodus-windows-amd64.zip) containing the `nexd.exe` and `nexctl.exe` executables. For simplicity, move those two files to `C:\` (or into a directory in your $PATH).
```

File: docs/quickstart.md [hunk 3]
```diff
@@ -52,6 +52,25 @@ Query the status of `nexd` and follow the instructions to register your device.
...
+- In the start menu type `cmd` and right click on `Command Prompt` and `Run as Administrator`. Then run the following in the command line and follow the URL to authenticate.
```
Comment (russellb) [inline docs/quickstart.md:61]:
```suggestion
- In the start menu, type `cmd` and right-click on `Command Prompt` and `Run as Administrator`. Then, run the following in the command line and follow the URL to authenticate.
```

File: docs/quickstart.md [hunk 4]
```diff
@@ -52,6 +52,25 @@ Query the status of `nexd` and follow the instructions to register your device.
...
+```terminal
+C:\Users\Administrator> cd C:\
+C:\> nexd.exe
+Your device must be registered with Nexodus.
+Your one-time code is: LYNW-HKGO
+Please open the following URL in your browser to sign in:
+https://auth.try.nexodus.io/realms/nexodus/device?user_code=LYNW-HKGO
+```
+
+- You will now have an ip address on an interface named `wg0` and be peered with other Nexodus devices in the organization which can be viewed by running `nexctl nexd peers list` in another window.
```
Comment (russellb) [inline docs/quickstart.md:72]:
```suggestion
- You will now have an IP address on an interface named `wg0` and be peered with other Nexodus devices in the organization, which can be viewed by running `nexctl nexd peers list` in another window.
```

**Review by russellb — COMMENTED** [review]
some minor suggestions - thanks for adding docs!

**Comment by nerdalert** [comment]
Thanks, added!

**Review by russellb — APPROVED** [review]

**Pull request merged**